src/selectors/layout.js

```diff
@@ -20,11 +38,13 @@ export const getGraphInput = createSelector(
     getVisibleLayerIDs,
     getOldgraphFlag,
     getFontLoaded,
+    getTriggerLargeGraphWarning,
   ],
-  (nodes, edges, layers, oldgraph, fontLoaded) => {
-    if (!fontLoaded) {
+  (nodes, edges, layers, oldgraph, fontLoaded, triggerLargeGraphWarning) => {
+    if (!fontLoaded || triggerLargeGraphWarning) {
```

Comment:
I suggest a similar thing here with the naming as earlier, for consistency.

Reply:
As I mentioned before, I don't particularly find the suggested naming to be of an enhancement worth enough for the efforts needed to refactor this?